Cargo.toml

```
@@ -33,5 +33,6 @@ chrono = "0.4.31"
 serde_yaml = "0.9.27"
 
 [features]
```

I think we also miss default here:

```suggestion
[features]
default = []
```

---

> I think we also miss default here:

Yes, I noticed that too. Initially, I thought not defining `default` was intentional. With your suggestion, I can proceed with the changes confidently.

After confirming the above comment with you, I will make the changes simultaneously

---

Yes, I think we can use this opportunity to introduce default